include/OpenSpaceToolkit/Astrodynamics/NumericalSolver.hpp

```diff
@@ -0,0 +1,261 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/// @project        Open Space Toolkit ▸ Astrodynamics
+/// @file           OpenSpaceToolkit/Astrodynamics/NumericalSolver.hpp
+/// @author         Antoine Paletta <antoine.paletta@loftorbital.com>
+/// @license        Apache License 2.0
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_NumericalSolver__
+#define __OpenSpaceToolkit_Astrodynamics_NumericalSolver__
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include <OpenSpaceToolkit/Physics/Time/Instant.hpp>
+#include <OpenSpaceToolkit/Physics/Time/Duration.hpp>
+#include <OpenSpaceToolkit/Physics/Units/Derived.hpp>
+
+#include <OpenSpaceToolkit/Core/Types/String.hpp>
+#include <OpenSpaceToolkit/Core/Types/Real.hpp>
+#include <OpenSpaceToolkit/Core/Types/Integer.hpp>
+
+#include <boost/numeric/odeint.hpp>
+#include <boost/bind.hpp>
```

Comment:
I need a quick memory refresher: do we expose boost anywhere else in OSTk's headers? Just checking that we're not setting a precedent here.

Reply:
I don't know, and I don't think we do. Is it bad to do this since boost is basically like c++ std?